src/main/webapp/index.html

```diff
@@ -6,7 +6,8 @@
     <link href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
     <title>smallcity</title>
   </head>
-  <body>
+  <!-- TODO: Move getGeolocation() function call to sensical location -->
```

Comment:
Don't let me forget to do this once we merge my branch into master, this may also create a small merge conflict for you.

Reply:
@ethanimooney  Perhaps create an issue to track this?